src/apps/create-patron-user-info/CreatePatron.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { useState, FC } from "react";
 import UserInfo from "./UserInfo";
+import { useConfig } from "../../core/utils/config";
 
 interface CreatePatronProps {
   userToken: string;
 }
 
 const CreatePatron: FC<CreatePatronProps> = ({ userToken }) => {
   const [cpr, setCpr] = useState<string | null>(null);
+  const config = useConfig();
+  const loginUrlConfig = config("loginUrlConfig");
```

Comment:
We are using the url hoc functionality for URLs in other places. Please use that

Reply:
I can, but then I would have to use the url, and get the href from that, which I thought was a little bit ugly. Should I do that?

Reply:
No you can just cast the URL object to string: `String(someUrlObject)`.